modules/geo-layers/src/tile-3d-layer/tile-3d-layer.js

```diff
@@ -227,18 +227,22 @@ export default class Tile3DLayer extends CompositeLayer {
 
   _makeSimpleMeshLayer(tileHeader, oldLayer) {
     const content = tileHeader.content;
-    const {attributes, modelMatrix, cartographicOrigin, texture} = content;
-    const {positions, normals, texCoords} = attributes;
+    const {modelMatrix, cartographicOrigin, texture} = content;
+    const attributes = {};
+    attributes.positions = content.attributes.positions;
+    attributes.positions.value = new Float32Array(attributes.positions.value);
```

Comment:
You are mutating `content.attributes.positions`

Reply:
Done